addons/stock/models/stock_move.py

```diff
@@ -874,8 +868,14 @@ def _get_relevant_state_among_moves(self):
             else:
                 return moves_todo[-1:].state or 'draft'
 
+    @api.onchange('product_id', 'picking_type_id')
+    def _onchange_product(self):
+        if self.product_id:
+            product = self.product_id.with_context(lang=self._get_lang())
+            self.description_picking = product._get_description(self.picking_type_id)
+
     @api.onchange('product_id')
-    def onchange_product_id(self):
+    def _onchange_product_id(self):
```

Comment:
why don't you merge this onchange with the one above ?

Reply:
No